doc/find-by-id.md

```diff
@@ -0,0 +1,21 @@
+# Find By Id
+
+When find an instance by its id, you can invoke the connector level method `findById` to use the lookup instead of query under the hood.
```

Comment:
```suggestion
To find an instance by its id, you can invoke the connector level method `findById` to use the lookup instead of query under the hood.
```

Can you please explain what are the ramifications of this approach? I believe the public implementation of `findById` is using a query to ensure that `Product.findById('123')` does not return a category model instance when `'123'` is an id of a category object in the database.

Reply:
@bajtos My bad, I forgot this PR :( 

There is a long story of why we chose to use connector level `findById`, the discussion is here: https://github.com/strongloop/loopback-connector-couchdb2/issues/46 and https://github.com/strongloop/loopback-datasource-juggler/pull/1459

tl;dr juggler's `findById` invokes `find` with filter, but doesn't call connector level's `findById` directly, and therefore doesn't have all the complicated processing logic before/after getting the connector's result.

But Cloudant has a much cheaper plan for query by id.